bin/deploy

```diff
@@ -255,12 +286,14 @@ main() {
 
     ARGS+=("--file" "$env_file")
 
+    echo "${ARGS[@]}"
     conda "${ARGS[@]}"
   )
 
   # intentionally outside of a lockfile subshell
   # shellcheck disable=SC1091
-  source activate "$LSST_CONDA_ENV_NAME"
+  echo "Activate environment ${LSST_CONDA_ENV_NAME}"
+  source activate "${LSST_CONDA_ENV_NAME}"
```

Comment:
Should we be considering changing to the official `conda activate` mechanism instead of this? https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment

Reply:
I think this is part of a bigger picture, like using a new conda distribution.

Reply:
We should definitely update our conda management (and support the new conda 4.6) but I'm happy for that to be a separate ticket (although if the change is only to change "source" to "conda" then we should do it rather than talking about it).

Reply:
I prefer a separate ticket.